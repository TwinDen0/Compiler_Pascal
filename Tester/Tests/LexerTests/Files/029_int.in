2147483647
21474836500