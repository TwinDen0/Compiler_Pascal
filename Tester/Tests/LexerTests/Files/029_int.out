1	1	INTEGER	2147483647	2147483647
(2,1) Fatal: Overflow symbol in integer
